pages.bn/android/dalvikvm.md

```diff
@@ -0,0 +1,8 @@
+# dalvikvm
+
+> অ্যান্ড্রয়েড জাভা ভার্চুয়াল মেশিন।
+> আরও তথ্য পাবেন: <https://source.android.com/devices/tech/dalvik>.
+
+- একটি নির্দিষ্ট জাভা প্রোগ্রাম শুরু করুন:
+
+`dalvikvm -classpath {{path/to/file.jar}} {{classname}}`
```

Comment:
Can you translate the placeholder here? As it will be useful for native speakers.